jmclient/jmclient/configure.py

```diff
@@ -986,6 +986,9 @@ def is_native_segwit_mode():
         return False
     return jm_single().config.get('POLICY', 'native') != 'false'
 
+def display_ypub_zpub():
+    return jm_single().config.get('POLICY', 'display_ypub_zpub') != 'false'
```

Comment:
If we have this setting (through I'm not sure, we could as well just always provide options for both xpub and ypub/zpub in GUI), it should also be added to default auto generated joinmarket.cfg.

Reply:
I'm not quite familiar with the generation of `joinmarket.cfg`, could you point it out for me where this is done in the code please?

Thanks!

Reply:
It's in `jmclient.configure.py` at the top as a string.

A point about that that's unobvious: that default config string is *always* loaded and then the contents of the user's configured `~/.joinmarket/joinmarket.cfg` are *overlaid* i.e. if a setting is not in the user's cfg file, the default will still exist.